PyRDF/RDataFrame.py

```diff
@@ -84,21 +84,26 @@ def get_num_entries(self):
         the given dataset.
 
         """
-        if isinstance(self.args[0], int):
+        first_arg = self.args[0]
+        if isinstance(first_arg, int):
             # If there's only one argument
             # which is an integer, return it.
-            return self.args[0]
+            return first_arg
+
+        elif isinstance(first_arg, ROOT.TTree):
+            return first_arg.GetEntries()
```

Comment:
To be consistent with the other comments: I would add a comment before this line: "The argument is a TTree or TChain, get the number of entries from it"